### PR TITLE
[Docs] Docs fixes for creating a new metricbeat module (#13233)

### PR DESCRIPTION
**Pull request by ioanatia**
Backport for https://github.com/elastic/beats/pull/13233

### CODE REVIEW

File: docs/devguide/create-module.asciidoc [hunk 1]
```diff
@@ -44,7 +44,7 @@ config file shipped by default.
 [float]
 ===== docs.asciidoc
 
-The `dosc.asciidoc` file contains the documentation about your module. During generation of the
+The `docs.asciidoc` file contains the documentation about your module. During generation of the
```
Comment (dedemorton) [inline docs/devguide/create-module.asciidoc]: thanks for catching this!

File: docs/devguide/create-metricset.asciidoc [hunk 2]
```diff
@@ -275,4 +275,3 @@ documentation.
 ==== What to Do Next
 This topic provides basic steps for creating a metricset. For more details about metricsets
 and how to extend your metricset further, see <<metricset-details>>.
-
```
Comment (dedemorton) [inline docs/devguide/create-metricset.asciidoc]: Add this blank line back in...it's sometimes required depending on how files are combined, so we always include a trailing blank line.

**Comment by ChrsMark** [comment]
Any updates on this @dedemorton ?

**Comment by dedemorton** [comment]
@ChrsMark I went ahead and merged this. I will backport it. Thanks!

**Comment by dedemorton** [comment]
Looks like this change was already made to later versions, so I'm removing the `needs_backport` label.

**Pull request merged**